near/nearcore
# refactor: remove unused RoutedMessageBody variant

Use a dummy to avoid spoiling the borsh. Add a test for this, while we
are at it.

Test Plan
-------

* add test for serialization format
* remove the variant, verify that the test fails
* re-install `Unused`, check that the test passes
